modules/RTC/TraceablePeerConnection.js

```diff
@@ -2443,21 +2406,10 @@ TraceablePeerConnection.prototype.addIceCandidate = function(
  * @returns {void}
  */
 TraceablePeerConnection.prototype.getStats = function(callback, errback) {
-    // TODO: Is this the correct way to handle Opera, Temasys?
     // TODO (brian): After moving all browsers to adapter, check if adapter is
     // accounting for different getStats apis, making the browser-checking-if
     // unnecessary.
-    if (browser.isTemasysPluginUsed()
-        || browser.isReactNative()) {
```

Comment:
Maybe this line being removed is why the tests fail?

Reply:
Oh, I missed that! Thanks for pointing out Lenny!